WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2047,6 +2047,16 @@ public void onAddMediaClicked() {
     }
 
     @Override
+    public void onTextDropped(String text) {
+        mEditorFragment.insertContent(text);
```

Comment:
Is it necessary to use `onTextDropped` here, making `EditPostActivity` part of the text insertion process? We could have `onDrag` directly call `execJavaScriptFromString("ZSSEditor.insertHTML(...);") and get rid of both `onTextDropped` and `insertContent` from `EditorFragmentAbstract`.